types/openfin/_v2/api/platform/golden-layout.d.ts

```diff
@@ -0,0 +1,887 @@
+declare namespace GoldenLayout {
+    export class GoldenLayout implements EventEmitter {
+        /**
+         * The topmost item in the layout item tree. In browser terms: Think of the GoldenLayout instance as window
+         * object and of root as the document.
+         */
+        root: ContentItem;
+
+        /**
+         * A reference to the (jQuery) DOM element containing the layout
+         */
+        container: JQuery;
+
+        /**
+         * True once the layout item tree has been created and the initialised event has been fired
+         */
+        isInitialised: boolean;
+
+        /**
+         * A reference to the current, extended top level config.
+         *
+         * Don't rely on this object for state saving / serialisation. Use layout.toConfig() instead.
+         */
+        config: Config;
+
+        /**
+         * The currently selected item or null if no item is selected. Only relevant if settings.selectionEnabled is set
+         * to true.
+         */
+        selectedItem: ContentItem;
+
+        /**
+         * The current outer width of the layout in pixels.
+         */
+        width: number;
+
+        /**
+         * The current outer height of the layout in pixels.
+         */
+        height: number;
+
+        /**
+         * An array of BrowserWindow instances
+         */
+        openPopouts: BrowserWindow[];
+
+        /**
+         * True if the layout has been opened as a popout by another layout.
+         */
+        isSubWindow: boolean;
+
+        /**
+         * A singleton instance of EventEmitter that works across windows
+         */
+        eventHub: EventEmitter;
+        _dragProxy: any;
+        dropTargetIndicator: any;
+
+        /**
+         * @param config A GoldenLayout configuration object
+         * @param container The DOM element the layout will be initialised in. Default: document.body
+         */
+        constructor(configuration: Config, container?: Element | HTMLElement | JQuery);
+
+        /*
+         * @param name 	The name of the component, as referred to by componentName in the component configuration.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/openfin/_v2/api/platform/golden-layout.d.ts;
  Line: 66;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.